### ISSUE TITLE
train setting

### ISSUE THREAD

**Issue by RuiboFan** [zzxslp/WCL#5]
Hello, I follow your training setting in run_mimic_cxr.sh, but I got the BLEU_1 score is 36. so what I shoud do to achieve you score in your paper. (I run your code on  1 V100)

**Comment by zzxslp** [comment]
Hi, sorry the default setting is used to finetune a pretrained report generator (i.e., a report generator X is already trained cross-entropy loss only so we have a meaningful representation space for contrastive learning, then we fine-tune the model  X with cross-entropy + contrastive loss to obtain the final model Y). Could you change contra_epoch in shell file to 10 or 15 and run the script again? Or run the script by loading a pretrained model X from [here](https://drive.google.com/file/d/1sbBOnvJMDarHhbg8WppyZPZvan5JnoyC/view?usp=sharing).

**Comment by RuiboFan** [comment]
Thank a lot, I try it again

**Comment by zzxslp** [comment]
Let me know if there are follow-up questions.
